.travis.yml

```diff
@@ -4,6 +4,13 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"
+
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
```

Comment:
Seems `sudo` is unnecessary:
https://travis-ci.org/python-security/pyt/builds/480490837/config